packages/holocron/src/ducks/compose.js

```diff
@@ -13,11 +13,21 @@
  */
 
 import { loadModule } from './load';
-import { LOAD_KEY } from '../constants';
+import { LOAD_KEY } from './constants';
 
 // This duck does not have a reducer
 export default null;
 
+function selectLoadModuleData(module) {
+  if (module && module.loadModuleData) {
```

Comment:
maybe put in some comments to remind us of what should be removed in the next major version

Reply:
Added in 924b611